Dockerfile

```diff
@@ -21,16 +21,16 @@ WORKDIR /src
 COPY README.md /microns-nda-access/.
 COPY Dockerfile /microns-nda-access/.
 COPY docker-compose.yml /microns-nda-access/.
-RUN git clone --single-branch --branch 21617_release https://github.com/cajal/microns_phase3_nda
+RUN git clone https://github.com/cajal/21617_data_release
```

Comment:
Do we still not want the --single-branch option to download only that branch? I guess it doesn't make much of a difference but it still makes sense to have imo